lib/Text/PRN/Slurp.pm

```diff
@@ -130,6 +129,7 @@ sub _from_io_handler {
             }
         }
         else {
+            chomp $row;
```

Comment:
@manwar So it could be an issue with test file (.prn file) ? because the first check is for header and it looks like after header no newline character, so it might be removing last character. What do you think ?